apps/misc/models.py

```diff
@@ -1,3 +1,19 @@
 from django.db import models
-
+from django.core.exceptions import ValidationError
 # Create your models here.
+
+class Initiative(models.Model):
+    title = models.CharField(max_length=50)
+    description = models.TextField(max_length=1000)
+    thumb_url = models.URLField(max_length=200,blank=True) 
+    thumbnail = models.ImageField(upload_to='misc/initiatives',blank=True)
+    
+    def save(self,*args, **kwargs): # if both thumbnail and thumb_url are absent, raise a validation error
+        if not self.thumb_url  and not self.thumbnail:
+            raise ValidationError('Atleast one field should be filled')   
```

Comment:
Throwing validation errors like this won't work. Yes, it shows the yellow screen with the error but only on the development environment, it throws 502 error when deployed on the server. So, you need to change this implementation.

I implement something similar in 3bcc2d5c, so you can refer to that too.